Ruby/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] It would be better if we have many-many/has-many-through relationship between a Person and a Book using the intermediate class Rental